apps/server/package.json

```diff
@@ -21,7 +21,11 @@
   "homepage": "https://github.com/Xemko/plants-app#readme",
   "dependencies": {
     "express": "^4.18.2",
+    "@types/node": "^18.0.0",
```

Comment:
W: in devDependencies is the right place. not need it here

Reply:
"dependencies": Packages required by your application in production.
"devDependencies": Packages that are only needed for local development and testing. 
I see.

Reply:
fixed